spaceship.zsh

```diff
@@ -300,15 +301,15 @@ spaceship_user() {
 # HOST
 # If there is an ssh connections, current machine name.
 spaceship_host() {
-  [[ $SPACESHIP_HOST_SHOW == false ]] && return
+  [[ $SPACESHIP_HOST_SHOW == false ]] && [[ $SPACESHIP_HOST_SHOW_ALWAYS == false ]] && return
 
-  [[ -n $SSH_CONNECTION ]] || return
-
-  _prompt_section \
-    "$SPACESHIP_HOST_COLOR" \
-    "$SPACESHIP_HOST_PREFIX" \
-    '%m' \
-    "$SPACESHIP_HOST_SUFFIX"
+  if [[ $SPACESHIP_HOST_SHOW_ALWAYS == true ]] || [[ -n $SSH_CONNECTION ]]; then
+    _prompt_section \
```

Comment:
Could you tell me what rule exactly i'm violating? Ive been looking and checking again and again and I just can't tell.

While you are at it. The colors in my next commit are always green (for both, ssh and not ssh), they don't seem to care for the default value i set. If I set a value in my .zshrc though it takes.